src/systems/systems/src/apu/mod.rs

```diff
@@ -126,23 +121,24 @@ impl<T: ApuGenerator, U: ApuStartMotor> AuxiliaryPowerUnit<T, U> {
         fire_overhead: &AuxiliaryPowerUnitFireOverheadPanel,
         apu_bleed_is_on: bool,
         apu_gen_is_used: bool,
+        bleed_air_valve: &mut impl ControllablePneumaticValve,
```

Comment:
Yeah, so I've played around with this a bit, but I couldn't really get anything working well. I tried to explain this in the comment here: 

> // TODO: Returning a mutable reference here is not great. I was running into an issue with the update order:
    // - The APU turbine must know about the bleed valve being open as soon as possible to update EGT properly
    // - To open the bleed valve, we need a signal from the ecb
    // - To get a signal from the ECB to open the bleed valve, we have to update the APU.
    // For now, we just pass over control of the bleed valve to the APU, so it can be updated after the ECB update but before the turbine update.